go.mod

```diff
@@ -1,27 +1,25 @@
 module github.com/numaproj/numaflow-go
 
-go 1.22
-
-toolchain go1.23.1
+go 1.24.1
```

Comment:
i lost the battle with `1.24` and `snyk` https://github.com/numaproj/numaflow-go/pull/177

Reply:
yeah too many CVEs, I reverted the changes.